reaktive-testing/src/commonMain/kotlin/com/badoo/reaktive/test/completable/TestAwait.kt

```diff
@@ -0,0 +1,32 @@
+package com.badoo.reaktive.test.completable
+
+import com.badoo.reaktive.completable.Completable
+import com.badoo.reaktive.completable.blockingAwait
+
+/**
+ * This method is used when you need to wait for an asynchronous operation to finish
```

Comment:
One of our users (@omainegra) had this use case. @CherryPerry What do you think about this?

Reply:
Looks good to me. However I'd say it looks like the TestScheduler should be preferred.

Reply:
Absolutely, in 99% of the cases. But there are some tricky cases when this may be needed.

Reply:
Yes, I usually prefer `TestScheduler`, but I was writing an Integration Test so I needed a real scheduler, and I couldn't block the test in JS

Reply:
@omainegra I also noticed that tests are executed on the main thread for e.g. iOS. So the following code will cause dead lock:
```kotlin
@Test
fun foo() =
    singleOf(1)
        .observeOn(mainThread)
        .testAwait()
```
In Android this code will fail since `Looper` is not instrumented.

So this is omething to keep in mind. The `mainScheduler` scheduler has to be overriden anyway.

Reply:
Good to know, thank you